lib/src/views/Clock/Clock.tsx

```diff
@@ -9,19 +10,19 @@ import { VIEW_HEIGHT } from '../../constants/dimensions';
 import { ClockViewType } from '../../constants/ClockType';
 import { PickerOnChangeFn } from '../../_shared/hooks/useViews';
 import { getHours, getMinutes } from '../../_helpers/time-utils';
-import { withDefaultProps } from '../../_shared/withDefaultProps';
+import { useDefaultProps } from '../../_shared/withDefaultProps';
```

Comment:
This is a step in the right direction. In hindsight, I wish I was more authoritarian in the past on pushing against `withDefaultProps` and `useStyles`.